docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -279,13 +279,13 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``ne``                      | P                               | ``level``                        |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``nlargest``                | N                               |                                  |                                                    |
+| ``nlargest``                | P                               |                                  | ``N`` if ``keep == "all"``                         |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
```

Comment:
forgot to mention that before, you also need to make sure you update dataframe.rst, series.rst to make sure those two APIs occur in the generated API doc

Reply:
Done